### DISCUSSION TITLE
Can any way refresh the new value of the sparkline?

### DISCUSSION THREAD

**Discussion by yangwawa0323** [vuetifyjs/vuetify#14349] [Q&A]
I want bind the value of sparkline and update the value for every 3000ms as well as process circular component.

`
 <v-progress-circular
                          rotate="360"
                          size="100"
                          width="15"
                          :value="circleValue"
                          color="teal"
                        >
                          {{ circleValue }}%
       </v-progress-circular>
` 

`
methods: {
    randomValue() {
      let _this = this;
      setInterval( ()=>{
        _this.circleValue = _.random(0,100)
      }, 3000)
    },
  },
`

**Comment by covik** [comment]
Method `randomValue()` should work properly, did you forget to call it? I assume you want to call it in `mounted()` hook.

Also, you don't need `_this` since you are using arrow function in `setInterval`.
See [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#call_apply_and_bind)

```js
methods: {
  randomValue() {
    setInterval( ()=>{ this.circleValue = _.random(0,100) }, 3000)
  },
},
```
